src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -68,4 +71,12 @@ public void notifyStaffForBulkPrintFailure(DocumentReadyToPrintEvent event) {
         );
         appInsights.trackEvent(BULK_PRINT_FAILED, event.getClaim().getReferenceNumber());
     }
+
+    private static Map<String, Object> wrapInMap(Claim claim) {
+        Map<String, Object> additionalData = new HashMap<>();
+        additionalData.put("caseReferenceNumber", claim.getReferenceNumber());
+        additionalData.put("letterType", claim.getLetterHolderId());
```

Comment:
whats the point in this?

Reply:
@rattur we do not want to mark print requests with letter holder ID. It does not make sense.

We just wanted to pass the type of the letter despite the fact there is only one CMC letter bulk printed ATM. Pass fix value like `defendantPack` or `defendantLetter` or similar.